components/org.wso2.carbon.appmgt.hostobjects/src/main/java/org/wso2/carbon/appmgt/hostobjects/APIStoreHostObject.java

```diff
@@ -234,29 +235,29 @@ public static NativeObject jsFunction_getBusinessOwner(Context cx, Scriptable th
             throw new AppManagementException("Invalid number of arguments. Arguments length should be one.");
         }
 
-        String appId = args[0].toString();
+        int businessOwnerId = Integer.valueOf(args[0].toString());
         APIConsumer apiConsumer = getAPIConsumer(thisObj);
-        BusinessOwner businessOwner = apiConsumer.getBusinessOwner(appId);
+        BusinessOwner businessOwner = apiConsumer.getBusinessOwner(businessOwnerId);
         NativeObject row = new NativeObject();
         row.put("businessOwnerId", row, businessOwner.getBusinessOwnerId());
         row.put("businessOwnerName", row, businessOwner.getBusinessOwnerName());
         row.put("businessOwnerEmail", row, businessOwner.getBusinessOwnerEmail());
         row.put("businessOwnerDescription", row, businessOwner.getBusinessOwnerDescription());
         row.put("businessOwnerSite", row, businessOwner.getBusinessOwnerSite());
-        Map<String, String> businessOwnerDetails = null;
-        businessOwnerDetails = businessOwner.getBusinessOwnerCustomProperties();
-        JSONObject businessOwnerDetailsObject = new JSONObject();
-        if(businessOwnerDetails != null) {
-            Set<String> keySet = businessOwnerDetails.keySet();
-            for (String key : keySet) {
-                businessOwnerDetailsObject.put(key, businessOwnerDetails.get(key));
-            }
-            row.put("businessOwnerDeatails", row, businessOwnerDetailsObject.toJSONString());
+        List<BusinessOwnerProperty> businessOwnerPropertiesList = businessOwner.getBusinessOwnerPropertiesList();
+        if(businessOwnerPropertiesList != null) {
```

Comment:
Already fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5